docs/AccessControl.md

```diff
@@ -24,6 +24,17 @@ convention is to:
 - promote readability and consistency among free-form names
 - interpret some typical data in the key-value style
 
+In short,
+1. `SRTO_STREAMID` is designed for a caller (client) to be able to identify himself, and tell what he wants.
```

Comment:
`SRTO_STREAMID` is designed for a caller (client) to be able to identify itself, and state what it wants.